### ISSUE TITLE
Topic items not translatable

### ISSUE THREAD

**Issue by matheusgomesms** [osm-fr/osmose-frontend#409]
Not sure if related to #214, #215, or to [osmose-frontend/tools/database/items_menu.txt](https://github.com/osm-fr/osmose-frontend/blob/master/tools/database/items_menu.txt), but many items in Topic section are not translatable (addr, amenity, bicycle, and so on).

First question: are these items supposed to be translatable?

If yes, then: can they be in alphabetical order in other languages (they are organized in English, so in other languages it is not ordered).

Also, where can I translate these strings? I saw [web/po](https://github.com/osm-fr/osmose-frontend/tree/master/web/po), but I guess it comes straight from Transifex, right?

Thanks for the great work!

**Comment by frodrigo** [comment]
> items_menu.txt, but many items in Topic section are not translatable (addr, amenity, bicycle, and so on).

All the content of menu item is send to po file and then to transifex.

The topic are hard coded osmose "tags". Translation is applied in the frontend, and only translated if the string match one other any translation os Osmose. If not intended translation.

But you are right. We may need to send topic tags to translation like we do for items_menu.txt.

**Comment by matheusgomesms** [comment]
Yes, the Topic list would be much more organized if all translatable. Thanks for the return, Frédéric!
